src/si/angle.rs

```diff
@@ -1,4 +1,4 @@
-//! Angle (dimensionless).
+//! Angle (unitless).
```

Comment:
Unitless isn't the appropriate description. https://jcgm.bipm.org/vim/en/1.8.html Let's use "quantity of dimension one" or "dimensionless quantity."